docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -225,6 +228,17 @@ property definition to the ``_meta.presto`` property of the target index mapping
         }
     }'
 
+Date types
+^^^^^^^^^^
+
+Elasticsearch supports a wide array of `date`_ formats including `built in date formats`_ and also `custom date formats`_.
+Currently, the elasticsearch connector supports neither built in date formats nor custom formats. Dates with the `format`_ property are ignored.
```

Comment:
Sounds confusing. If it supports neither then what is actually supported?

It's generally better to word these things as the supported case. i.e. `Currently the elasticsearch connector supports only xyz dates. All other date formats including built-in or custom are not supported.`?

Reply:
Oh yeah. When I re-read it, even I felt like confusing. Let me rephrase it.
`Currently, the elasticsearch connector supports only default date type. All other date formats including built-in and custom formats are not supported. Date with the `format`_ property is ignored.`

Reply:
Yes it does. Thanks.